plan/plan.go

```diff
@@ -146,6 +146,10 @@ func (p *Plan) Calculate() *Plan {
 
 	for _, topRow := range t.rows {
 		for _, row := range topRow {
+			if endpoint.EndpointsHaveEmptyTargets(row.candidates) {
```

Comment:
How about adding a comment here
```suggestion
                        // We ignore defined Endpoints with unknown targets, as there seems to be
                        // intention for the DNS entry, but it is not known where to point them.  
			if endpoint.EndpointsHaveEmptyTargets(row.candidates) {
```